gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.0.1-all.zip
```

Comment:
I created this PR 5 days ago, there was no `6.1`
What prevents me to have a `6.0.1` when this will be reviewed again ?

That's why I vote to approve and merge this, to go ahead.
And for sure I will create a follow up new PR with 6.1

Reply:
Well, not really, `6.1` was released one day before this PR was created. https://gradle.org/releases/

Reply:
The reality is always faster: there is a `6.1.1` 
But it's pointless to block this PR because of this

Reply:
`6.1.1` has been released after this PR creation while `6.1` was released before this PR creation, as I said in my comment above. That's why I suggested to use directly `6.1` instead of `6.0.1`. I already applied the change yesterday (here and in https://github.com/owncloud/android/pull/2792) and the PRs are ready to QA since then.